python/langsmith/_internal/_serde.py

```diff
@@ -35,14 +31,8 @@ def _simple_default(obj):
         # https://github.com/ijl/orjson#serialize
         if isinstance(obj, datetime.datetime):
             return obj.isoformat()
-        if isinstance(obj, uuid.UUID):
+        elif isinstance(obj, uuid.UUID):
             return str(obj)
-        if hasattr(obj, "model_dump") and callable(obj.model_dump):
-            return obj.model_dump()
-        elif hasattr(obj, "dict") and callable(obj.dict):
-            return obj.dict()
-        elif hasattr(obj, "_asdict") and callable(obj._asdict):
-            return obj._asdict()
```

Comment:
removed these are they are dead code - namedtuple, and pydantic.BaseModel are handled in `_serialize_json`